test/common/testlib.py

```diff
@@ -1125,6 +1059,35 @@ def join_some(n):
             sys.stdout.write("# TESTS PASSED {0}\n".format(details))
         return count
 
+    def filterOutput(self, test, failed, output):
+        # Check how many retries we can do of this test
+        tries = getattr(test, "retryCount", 0)
+        tries += 1
+        setattr(test, "retryCount", tries)
+
+        # Didn't fail, just print output and continue
+        if tries >= 3 or not failed:
+            sys.stdout.write(output)
+            return failed, False
+
+        # Otherwise pass through this command if it exists
+        cmd = [ "tests-policy", testvm.DEFAULT_IMAGE ]
+        try:
+            proc = subprocess.Popen(cmd, stdin=subprocess.PIPE, stdout=subprocess.PIPE)
+            (output, error) = proc.communicate(output)
```

Comment:
`error` will always be None as it's not redirected. This can be done more simply with `output = subprocess.check_output([ "tests-policy", testvm.DEFAULT_IMAGE ])`.

Reply:
Not sure how we would pass the traceback to the command in that case. Is there a cool way to pass input to ```check_output```?

Reply:
There isn't, nevermind me.